src/devtools/views/left-rail/index.js

```diff
@@ -29,21 +36,50 @@ class ActionList extends Presenter {
     )
   }
 
+  initDrag = e => {
+    startX = e.clientX
+    startWidth = this.state.paneWidth
+
+    document.documentElement.addEventListener('mousemove', this.onDrag, false)
+    document.documentElement.addEventListener('mouseup', this.stopDrag, false)
+  }
+
+  onDrag = e => {
+    this.setState({
+      paneWidth: startWidth + (e.clientX - startX)
+    })
+  }
+
+  stopDrag = e => {
+    document.documentElement.removeEventListener(
+      'mousemove',
+      this.onDrag,
+      false
```

Comment:
Do you need false? I thought passing `true` was only needed for capture.

I _think_ it can be omitted:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener